mongoDB.go

```diff
@@ -113,6 +117,47 @@ func mapUniqueRecords(records *[]RecordLink) map[string]aggregatedRecord {
 	return recordMap
 }
 
+// Attempts to read the geometry from the Postgres table and update to Mongo
+func findElementPoints(s *Server, session *mgo.Session, record *RecordLink, tableMap map[int]TableMeta) {
+	// Only update if no lat or lon information stored.
+	if record.Lat < 0.1 && record.Lat > -0.1 && record.Lon < 0.1 && record.Lon > -0.1 {
```

Comment:
`record.Lat` and `record.Lon` are both `float64` types, which cannot be validated against 0 as they sometimes end up being 0.000000000000000000000034 or similar.  This check treats any very small value as 0.  Do you have any other suggestions?